source-code/sdk-js/src/adapter-sveltekit/ast-transformation/tests/hooks.server.js.test.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, it } from 'vitest'
```

Comment:
Please colocate the test files next to the implementation to be consistent with the rest of the code base.